### PR TITLE
Allow relative time values in CSRF-cookie

### PR DESCRIPTION
**Pull request by Schlaefer**
Refs #6889

### CODE REVIEW

File: src/Controller/Component/CsrfComponent.php [hunk 1]
```diff
@@ -120,12 +121,14 @@ public function implementedEvents()
      */
     protected function _setCookie(Request $request, Response $response)
     {
+        $expires = new Time($this->_config['expiry']);
```
Comment (markstory) [inline src/Controller/Component/CsrfComponent.php]: Will this cause issues with any old integer based config values? Or were they always broken/wrong?

Comment (Schlaefer) [inline src/Controller/Component/CsrfComponent.php]: Didn't crawl through the code, but CookieComponent::_write() uses the same way to set the unix timestamp as string for "expire" on the Response::cookie().

So I assume we're fine.

Comment (Schlaefer) [inline src/Controller/Component/CsrfComponent.php]: `Network\Response::_setCookies()` uses PHP's `setcookie()`. The "expire" value gets to `setcookie()` as string after putting it through `Time()->format('U')`, but casts it automatically to int. So it should be O.K.

**Comment by markstory** [comment]
documentation updated in cakephp/docs@d505072521e2716bea2803494d2ac838761bc784

**Pull request merged**